Mining Droid + Borg Tank Tread Movement

Vivinet opened the pull request:
- Adds the mining droid sprite to the default mining borg module selection (Original icon from baystation, colored to be more fitting with lavaland).

- Adds movement states for both security and engineering tank tread sprites.

- Fixed static state security tread sprite.

ProfessorDonut committed:
Mining Droid + Borg Tank Tread Movement

- Adds the mining droid sprite to the default mining borg module selection.

- Adds movement states for both security and engineering tank tread sprites.

- Fixed static security tread state sprite.

deathride58 reviewed (changes requested):
You should be able to revert the changes to the robots.dmi in icons/mob/, and move all of the borg module sprites you touched there to robots.dmi in modular_citadel/icons/mob. Just be sure to add cyborg_icon_override values to the affected sprites.

ProfessorDonut committed:
Requested changes and then some

- Moved all modified sprites to modular_citadel, made overrides for sec-tread and eng-tread, reverted original robots.dmi

- Added open cover states to modular_citadel robots.dmi

- Removed light found on normal engi-borg sprite that was not on the tread version.

Commit:
Update admin.txt

Commit:
Update admin.txt

deathride58 reviewed (approved):
your coder!!!!
